whats-new-panel.yaml

```diff
@@ -174,7 +96,42 @@
     || (messageImpressions['WHATS_NEW_BADGE_82']|length >= 1 && currentDate|date -
     messageImpressions['WHATS_NEW_BADGE_82'][0] <= 345600000)) &&
     "browser.messaging-system.whatsNewPanel.enabled"|preferenceValue &&
-    region == "US"
+    localeLanguageCode != 'fr'  # Because WNP_MEDIA message is not yet translated
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_DNS_OVER_HTTPS
```

Comment:
Normally, we'd end the id with a browser version, shall we stick to that convention?